### PR TITLE
homework #1

### PR DESCRIPTION
**Pull request by captain2b**

### CODE REVIEW

File: tasks/task1.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    input: process.stdin,
+    output: process.stdout,
+    terminal: false,
+});
+
+rl.setPrompt('Please insert string: \n');
+rl.prompt();
+rl.on('line', (s) => {
+    console.log('Reversed string:');
+    console.log(`${[...s].reverse().join("")} \n`);
```
Comment (farafonoff) [inline tasks/task1.js]: more readable: s.split('').reverse().join('')

File: tasks/task2.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+let lineCount = 0;
+
+const onError = (err, module = 'parsing') => {
+    console.log(module ? `Error while ${module}: ${err}` : err);
+}
+const onComplete = () => {
+    console.log('Transformation completed');
+    readable.destroy();
+}
+writable
```
Comment (farafonoff) [inline tasks/task2.js:14]: please look at https://prettier.io/ and https://eslint.org/ to make code style more consistent

File: tasks/task2.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import csv from 'csvtojson';
+
+const inputFileURL = './csv/nodejs-hw1-ex1.csv';
+const outputFileURL = 'nodejs-hw1-ex1.txt';
+
+const readable = fs.createReadStream(inputFileURL, 'utf8');
+const writable = fs.createWriteStream(outputFileURL);
+let lineCount = 0;
+
+const onError = (err, module = 'parsing') => {
```
Comment (farafonoff) [inline tasks/task2.js]: function is far away from usage. More verbose name (onCsvParseError) would be helpful

File: tasks/task2.js (hunk 2 above)
Comment (captain2b) [reply tasks/task2.js:14]: sure. sorry, new IDE
added eslint

**Pull request merged**